content/en/blog/_posts/2021-12-16-kubernetes-is-moving-on-from-dockershim.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: blog
+title: "Kubernetes is moving on from Dockershim: commitments and next steps"
+date: 2021-12-16
+slug: kubernetes-is-moving-on-from-dockershim
+---
+
+**Authors:** Sergey Kanzhelev (Google), Jim Angel (Google), Davanum Srinivas (VMware), Shannon Kularathna (Google), Chris Short (AWS), Dawn Chen (Google)
+
+Kubernetes is removing dockershim in upcoming version 1.24. Please welcome new
+container runtimes with more exciting features, a smaller kubelet, and faster
+engineering velocity for new features. If you [use Docker as a Container Runtime](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/)
+for Kubernetes, get ready to migrate from dockershim! To check if you're
+affected, refer to [Check whether dockershim deprecation affects you](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/).
+
+## Why we’re moving away from dockershim
+
+Docker was the first container runtime used by Kubernetes. This is one of the
+reasons why Docker is so familiar to many Kubernetes users and enthusiasts.
+Kubernetes uses dockershim as a container runtime interface (CRI) to let system
+components (like the kubelet) talk to Docker. It was built into Kubernetes code.
+Removing dockershim inevitably affects the entire CNCF family of projects beyond
+Kubernetes. Dependencies on Docker and dockershim have crept into various tools
+and projects in the Kubernetes ecosystem.
+
+There are a few container runtimes today that are often lighter, faster and more
+secure than when Docker was the only choice. By removing the dockershim CRI,
+we're embracing the first value of CNCF: "[Fast is better than slow](https://github.com/cncf/foundation/blob/master/charter.md#:~:text=Fast%20is%20better%20than%20slow)".
+Migrating to another runtime will ensure that applications are more portable,
+with fewer dependencies on the infrastructure. Your apps will be more secure,
+with new features unlocked. A different runtime will also let us develop
+features like cgroupv2 support faster.
+
+## Deprecation timeline
+
+We announced dockershim deprecation in December 2020.  Full removal is targeted
+to Kubernetes 1.24, which is planned for April 2022. This deprecation timeline
+aligns with our [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior),
+which states that deprecated behaviors must function for no less than 1 year
+after their announced deprecation. We'll support version 1.23, which has
+dockershim, for another year in the Kubernetes project (and likely longer with
+managed Kubernetes providers). You'll have time to migrate. If you have more
```

Comment:
```suggestion
```suggestion
dockershim, until its [end of life](/releases/#release-v1-23), February 28, 2023. 
Managed Kubernetes providers will likely provide longer support periods.
If you have more
```